.github/ci_templates/tests.yml

```diff
@@ -7,8 +7,8 @@ docker_options: '-w /deckhouse -v ${{env.HOME}}/go-pkg-cache:/go/pkg'
 
 {!{ define "matrix_run_args" }!}
 # <template: matrix_run_args>
-args: 'go test -timeout=${{env.TEST_TIMEOUT}} ./testing/matrix/ -v'
-docker_options: '-w /deckhouse -v ${{env.HOME}}/go-pkg-cache:/go/pkg'
+args: 'sh -c "rm -f ee/modules/030-cloud-provider-openstack/candi ee/modules/030-cloud-provider-vsphere/candi; go test -timeout=${{env.TEST_TIMEOUT}} ./testing/matrix/ -v ; ln -s /deckhouse/candi/cloud-providers/openstack/ ee/modules/030-cloud-provider-openstack/candi ; ln -s /deckhouse/candi/cloud-providers/vsphere/ ee/modules/030-cloud-provider-vsphere/candi"'
```

Comment:
Restoring links with `ln -s` is not needed, each job has its own copy of repository.

But what is the problem with `ee/modules/*/candi`?

Reply:
When we try to run tests with simlinks, we get an error - 
```
=== RUN   TestMatrix/cloud-provider-openstack
panic on linter run occurred: chart load: error evaluating symlink /deckhouse/ee/modules/030-cloud-provider-openstack/candi: lstat /Users/Ash/work/flant/deckhouse-oss/candi/cloud-providers/openstack: no such file or directory
=== RUN   TestMatrix/cloud-provider-vsphere
panic on linter run occurred: chart load: error evaluating symlink /deckhouse/ee/modules/030-cloud-provider-vsphere/candi: lstat /Users/Ash/work/flant/deckhouse-oss/candi/cloud-providers/vsphere: no such file or directory
```
Helm chart render lib cannot parse inconsistent synlinks.